user-guide/Advanced_Functionality/Swarming/Tutorials/EnableSwarmingTutorial.md

```diff
@@ -0,0 +1,175 @@
+
+1. Open the *Mask Critical Alarms* script in the Automation app, scroll down, and click *Validate* to check for errors and warnings.
+
+   You will see that a warning is detected.
+
+   ![AlarmID Usage Problem Warning](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage_Problem_Warning.png)
+
+   Note that you could also open the script via DIS in Visual Studio instead, if you have this installed. In that case, if you target the *Skyline.DataMiner.Dev.Automation* package version 10.5.1 or higher, you will also see an obsolete warning for this script.
+
+1. Update the code in the *foreach* block (line 85) as follows:
+
+   Before:
+
+   ```csharp
+      var maskRequest = new SetAlarmStateMessage();
+      maskRequest.DataMinerID = alarm.DataMinerID;
+      maskRequest.AlarmId = alarm.AlarmID;
+      maskRequest.DesiredStatus = AlarmUserStatus.Mask;
+      maskRequests.Add(maskRequest);
+    ```
+
+   After:
+
+   ```csharp
+      var maskRequest = new SetAlarmStateMessage(alarm.TreeID, AlarmUserStatus.Mask, "");
+      maskRequests.Add(maskRequest);
+   ```
+
+1. Click *Validate* again to see if all warnings and errors are gone.
+
+   ![Alarm ID usage problem fix](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage_Problem_Fix.png)
+
+1. Click *Execute* to run the script and test if it still works correctly.
+
+1. If any other scripts or protocols are shown in the alarm ID usage summary, update these in a similar way to make sure they refer to the tree ID or alarm tree ID and validate and test them.
+
+   For detailed information, refer to [Preparing scripts and connectors for Swarming](xref:SwarmingPrepare)
+
+1. Run the the *Enable Swarming* script again and click the *Analyze* button again.
+
+   If you have indeed correctly updated all scripts and protocols, a new button will be displayed at the bottom of the window. If you do not see this button, check which scripts or protocols still need to be updated and adjust them as mentioned above.
+
+   ![AlarmID Usage Problem Fix](~/user-guide/images/Swarming_Tutorial_Enable_No_Problems.png)
+
+1. Click *Restart DMS and enable Swarming*.
+
+   The script will now double-check the requirements (which will again take some time), enable Swarming, and restart DataMiner.
+
+   > [!IMPORTANT]
+   > This will restart all DataMiner Agents in your DMS.
+
+## Step 5: Swarm your first element
+
+Once DataMiner has restarted, you can test the Swarming functionality by moving an element from one host to another host in your DMS. You will only be able to follow these steps if you are using a DataMiner System consisting of multiple Agents.
+
+1. Go to *System Center* > *Agents* > *Status* and click the *Swarm* button in the lower right corner.
+
+   If you see a *Migrate* button here instead, this means Swarming has not been successfully enabled.
```

Comment:
Is it worth mentioning that if you see no button at all, you are on a single agent? (so not a DMS with other agents)